doc/source/api_reference/models.rst

```diff
@@ -14,6 +14,17 @@ the Global Coefficients and the Domain of Analysis, and its project id. The
 object is, then, parsed to :py:meth:`models.build()<ModelDirectory.build>` for
 launching a build.
 
+DomainAxisDefinition
+--------------------
+
+.. autoclass:: DomainAxisDefinition()
+
+
+DomainOfAnalysis
+--------------------
+
+.. autoclass:: DomainOfAnalysis()
+
```

Comment:
1. I renamed to page to `Models`
2. Since the classes are still together in the same file, it's more coherent to have them all in the same page. I set the Directory and the Model first to be aligned with the rest of the API reference. Nevertheless, once more classes are introduced to cover the complete configuration (which is coming in the next PR), it'd make sense to put them in another file, and, thus, reflect that to the documentation.